pkg/apis/core/v1/conversion.go

```diff
@@ -286,6 +286,22 @@ func Convert_core_PodStatus_To_v1_PodStatus(in *core.PodStatus, out *v1.PodStatu
 	return nil
 }
 
+func Convert_core_SecurityContext_To_v1_SecurityContext(in *core.SecurityContext, out *v1.SecurityContext, s conversion.Scope) error {
+	if err := autoConvert_core_SecurityContext_To_v1_SecurityContext(in, out, s); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func Convert_core_PodSecurityContext_To_v1_PodSecurityContext(in *core.PodSecurityContext, out *v1.PodSecurityContext, s conversion.Scope) error {
+	if err := autoConvert_core_PodSecurityContext_To_v1_PodSecurityContext(in, out, s); err != nil {
+		return err
+	}
+
+	return nil
+}
+
```

Comment:
why was this needed?

Reply:
This was automatically generated and I wasn't sure whether it was needed. Now it is removed.